starter-kits/product/_overview_2.md

```diff
@@ -1,6 +1,6 @@
 % ## Cite this product
-
-% > Your citation
+% - required
 
 % ## Publications
```

Comment:
Minor thing: Since this is required, the field shouldn't be commented out.

Reply:
Fixed.